ocrd_segment/extract_lines.py

```diff
@@ -103,39 +109,20 @@ def process(self):
             worksheet.write('B1', 'Text', bold)
             worksheet.write('C1', 'Status', bold)
             worksheet.write('D1', 'Image', bold)
-            worksheet.write('E1', 'ſ ꝛ aͤ oͤ uͤ æ œ Æ Œ ℳ  ç ę ë - ⸗ = Α α Β β ϐ Γ γ Δ δ Ε ε ϵ Ζ ζ Η η Θ θ ϑ Ι ι Κ κ ϰ Λ λ Μ μ Ν ν Ξ ξ Ο ο Π π ϖ Ρ ρ ϱ Σ σ 	ς ϲ Τ τ Υ υ ϒ Φ φ ϕ Χ χ Ψ ψ Ω ω')
+            symbols = 'ſ ꝛ aͤ oͤ uͤ æ œ Æ Œ ℳ  ç ę ë - ⸗ = Α α Β β ϐ Γ γ Δ δ Ε ε ϵ Ζ ζ Η η Θ θ ϑ Ι ι ' \
+                      'Κ κ ϰ Λ λ Μ μ Ν ν Ξ ξ Ο ο Π π ϖ Ρ ρ ϱ Σ σ ς ϲ Τ τ Υ υ ϒ Φ φ ϕ Χ χ Ψ ψ Ω ω'.split(' ')
+            for i, s in enumerate(symbols):
+                col_idx = 3 + i
+                worksheet.write_string(0, col_idx, s)
+                worksheet.set_column(col_idx, col_idx, 2)
```

Comment:
Good idea! Will the line images still stretch across all these columns if necessary?

Reply:
Yes, the line image will stretch at least in my tests in Excel 2016 and LibreOffice 5. I had also prepared a solution with macros assigned to buttons so that you just need to click a "virtual key" to copy the glyph into clipboard. But macros are a source of vulnerabilities and it's hard to to argue for enabling them.

Another idea was to have a dead simple HTML+Javascript page with a virtual keyboard of click-to-copy symbols to have open in a split browser window.

But let's first test how this "one-char-per-cell" works, ergonomically.

Reply:
Absolutely!